IAMOTZ/whois
# Add unit tests to the codebase

### What does this PR do
- add unit tests to the attachment functions
- add unit tests to the `Andelan` class

### How should it manually tested
- Clone and set up the project
- Check out to the branch containing the changes in this PR
- Run `npm test` to execute the tests. All the tests should pass and the coverage report should display afterwards.
